internal/pkg/build/sources/conveyorPacker_oci.go

```diff
@@ -318,6 +334,18 @@ func (cp *OCIConveyorPacker) insertRunScript() (err error) {
 		}
 	}
 
+	if len(cp.imgConfig.WorkingDir) > 0 {
```

Comment:
That may be an issue because OCI engine or Singularity couldn't override current working directory with the one user provided, may be better to let engines decides by parsing image config

Reply:
Good point, parsing the image config would be a better way to implement `CMD` and `ENTRYPOINT` as well. I'll remove this from this PR for now